attitude_error_mon/att_err_mon.py

```diff
@@ -0,0 +1,354 @@
+#!/usr/bin/env python
+
+import os
+import argparse
+import jinja2
+import numpy as np
+import matplotlib
+
+if __name__ == "__main__":
+    matplotlib.use("Agg")
+import matplotlib.pyplot as plt
+
+from astropy.table import Table, vstack, Column
+from astropy.io import ascii
+from Ska.Matplotlib import plot_cxctime
+from Chandra.Time import DateTime
+from kadi import events
+from Ska.engarchive import fetch
+from cxotime import CxoTime
+
+
+FILE_DIR = os.path.dirname(os.path.abspath(__file__))
```

Comment:
```suggestion
FILE_DIR = Path(__file__).parent
```
Note that `__file__` is always an absolute path since Python 3.9. https://docs.python.org/3/whatsnew/3.9.html#other-language-changes